pkg/harness/harness.go

```diff
@@ -1,27 +1,80 @@
 package harness
 
 import (
-	"github.com/giantswarm/micrologger"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	yaml "gopkg.in/yaml.v2"
 )
 
-type Harness struct {
-	logger micrologger.Logger
+// Status represents the current state of the
+// test harness. It is the token passed amongst
+// tasks.
+type Status struct {
+	RemoteCluster bool   `yaml:"remoteCluster"`
+	ImageTag      string `yaml:"imageTag"`
+}
+
+// Init initializes the harness
```

Comment:
Since those functions are of `Task` type I'd like to see comments like `Init is a Task initializing the harness.`. This would make it easier to realise that. Also please try add full stops at the end <3

Reply:
Done